src/styles/pages/_live.scss

```
@@ -285,6 +286,13 @@
   &:focus {
     outline: 0;
     opacity: .9;
+    text-decoration: none;
+  }
+
+  &:focus {
+    .w-event-carousel__description {
+      outline: 1px solid $FOCUS_COLOR;
```

```suggestion
      box-shadow: inset 0 0 0 1px $FOCUS_COLOR;
```

I think this helps with z clipping on the left edge of day 1